llvm/docs/LangRef.rst

```diff
@@ -19956,6 +19956,36 @@ the follow sequence of operations:
 
 The ``mask`` operand will apply to at least the gather and scatter operations.
 
+'``llvm.experimental.vector.masked.extract.last.active``' Intrinsic
```

Comment:
It was due to a conflict in the AutoUpgrade code, where it matched `llvm.experimental.vector.extract` and was converted to the wrong intrinsic. I've changed the updater now to avoid the problem.